lib/guide-view.js

```diff
@@ -82,6 +83,25 @@ export default class GuideView {
               </div>
             </details>
 
+            <details className='welcome-card' {...this.getSectionProps('teletype')}>
+              <summary className='welcome-summary icon icon-radio-tower'>
+                Real time collaboration with <span class='welcome-highlight'>Teletype</span>
```

Comment:
How would you feel about rephrasing this as a call-to-action? Maybe something like:

> Collaborate in real time with Teletype

That would make it more consistent with most of the other sections (e.g., "Install a Package", "Hack on the Init Script", etc.). At the moment, the "Version control with <span class='welcome-highlight'>Git and GitHub</span>" item is the only one that doesn't use a call-to-action phrasing.

Reply:
Good point, I will update the title and merge this pull-request. Thanks, @jasonrudolph!